frontend/src/components/Invest/LoanList/LoanList.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div class="loan-list">
+    <div class="ll-title"> Loan listing </div>
+    <no-loans v-if="!loans" />
+    <div v-else class="ll-container">
+      <md-table md-sort="procent-needed">
+        <md-table-header>
+          <md-table-row>
+            <md-table-head md-sort-by="name" class="ll-th"> NAME </md-table-head>
+            <md-table-head md-sort-by="score" class="ll-th"> SCORE </md-table-head>
+            <md-table-head md-sort-by="amount-needed" class="ll-th"> AMOUNT NEEDED </md-table-head>
+            <md-table-head md-sort-by="time" class="ll-th"> TIME </md-table-head>
+            <md-table-head md-sort-by="procent-funded" class="ll-th"> % FUNDED </md-table-head>
+            <md-table-head md-sort-by="procent-needed" class="ll-th"> % NEEDED </md-table-head>
+          </md-table-row>
+        </md-table-header>
+        <md-table-body>
+          <md-table-row @click.native='goToLoan(loan.id)' v-for="loan in loans" :key="loan.id">
+            <md-table-cell class="ll-td">{{ loan.userName }}</md-table-cell>
+            <md-table-cell class="ll-td ll-score"> <user-score :value="loan.userScore"/> </md-table-cell>
+            <md-table-cell class="ll-td">{{ loan.amountNeeded }}</md-table-cell>
+            <md-table-cell class="ll-td ll-time">{{ loan.time }}</md-table-cell>
+            <md-table-cell class="ll-td">{{ loan.procentFunded }}</md-table-cell>
+            <md-table-cell class="ll-td">{{ loan.procentNeeded }}</md-table-cell>
+          </md-table-row>
+        </md-table-body>
+      </md-table>
+    </div>
+  </div>
+</template>
+
+<script>
+import NoLoans from './NoLoans';
+import UserScore from '../../common/UserScore';
```

Comment:
Is there a way to do imports in JS that doesn't involve a ton of '../../../../'? Some sort of absolute imports?

Reply:
Yes:
```
import UserScore from '@/components/common/UserScore'
```
I will aviod: "../.../../../../"